web/views/layout_view.ex

```diff
@@ -11,4 +11,14 @@ defmodule CoursePlanner.LayoutView do
   def get_program_name do
     Settings.get_value("PROGRAM_NAME")
   end
+
+  def render_user_role_based_navbar(%{assigns: %{current_user: current_user}} = conn) do
+    case current_user.role do
```

Comment:
As I see the only difference in the cases is data related, do what do you think of extracting the data to a map? for  example
```elixir
@navbars %{"Coordinator" => "coordinator_app_navbar.html", ... }
def render_user_role_based_navbar(%{assigns: %{current_user: current_user}} = conn) do
  render(@navbars[current_user.role], conn:conn)
end
```